be/tool/beck.c

```diff
@@ -1566,6 +1595,168 @@ static void builder_fini(struct builder *b)
 	       b->b_act, b->b_tx);
 }
 
+static void override_be_cfg_defaults(struct m0_be_domain_cfg *cfg,
+			     	     const char              *str_key,
+			     	     const char              *str_value)
+{
+	uint64_t  value1_64;
+	uint64_t  value2_64;
+	char     *s1;
+	char     *s2;
+	bool      value_overridden = true;
+
+	if (m0_streq(str_key,"bec_tx_active_max"))
+		cfg->bc_engine.bec_tx_active_max = m0_strtou64(str_value, 0, 10);
+	else if (m0_streq(str_key, "bec_group_nr"))
+		cfg->bc_engine.bec_group_nr = m0_strtou64(str_value, 0, 10);
+	else if (m0_streq(str_key, "tgc_tx_nr_max"))
+		cfg->bc_engine.bec_group_cfg.tgc_tx_nr_max = m0_strtou64(str_value,
+									 0, 10);
+	else if (m0_streq(str_key, "tgc_size_max")) {
+		s1 = m0_strdup(str_value);
+		s2 = strchr(s1, ',');
+
+		*s2 = '\0';
+		s2++;
+
+		value1_64 = m0_strtou64(s1, 0, 10);
+		value2_64 = m0_strtou64(s2, 0, 10);
+		m0_free(s1);
+
+		cfg->bc_engine.bec_group_cfg.tgc_size_max = 
+					M0_BE_TX_CREDIT(value1_64, value2_64);
+	}
+	else if (m0_streq(str_key, "tgc_payload_max"))
+		cfg->bc_engine.bec_group_cfg.tgc_payload_max = 
+						m0_strtou64(str_value, 0, 10);
```

Comment:
```
uint64_t m0_strtou64(const char *str, char **endptr, int base);
```
m0_strtou64(str_value, 0, 10);

The second argument should be NULL, according to the coding-style.